tests/FakeItEasy.Specs/ArgumentCapturingSpecs.cs

```diff
@@ -283,7 +283,12 @@ public static void CaptureNegatedConstrainedValues(Action<int> fake, Captured<in
 
             "Then the captured arguments instance's values match the input at the time of the calls"
                 .x(() => capturedArgument.Values.Should().BeEquivalentTo(

```

Comment:
You replaced the other `BeEquivalentTo`s with `Equal`, but not this one. Is it intentional?
(I suspect it doesn't work with nested arrays, just checking)

Reply:
It didn't work with nested arrays.